### PR TITLE
Printer copies

### PR DESCRIPTION
**Pull request by jeanpaulrobineau**
Hello,

This module will allow to specify per report or user/report the number of copies.
This is inspired by printer_tray module

kind regards,
Jean-Paul Robineau

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please rename module to printer_copy_number, for not using plurals in module name as stated in OCA guidelines.

**Comment by jeanpaulrobineau** [comment]
I renamed it

**Comment by pedrobaeza** [comment]
Thanks!

**Comment by thomich** [comment]
great feature.. lets merge it ;)

File: printer_copy_number/models/res_users.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    copies = fields.Integer(
```
Comment (Garamotte) [inline printer_copy_number/models/res_users.py:9]: I would prefer this field to be named `print_copy_count` or something like that, as the users model is very generic, `copies` could be anything.

File: printer_copy_number/views/ir_actions_report_xml.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<openerp>
+  <data>
```
Comment (Garamotte) [inline printer_copy_number/views/ir_actions_report_xml.xml:3]: Replace `openerp` and `data` nodes by a single `odoo` node.

File: printer_copy_number/views/printing_report_xml_action.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data noupdate="0">
```
Comment (Garamotte) [inline printer_copy_number/views/printing_report_xml_action.xml:3]: Replace `openerp` and `data` nodes by a single `odoo` node.
Specifying `noupdate="0"` is not needed (but not a problem).

File: printer_copy_number/views/res_users.xml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<openerp>
+  <data>
```
Comment (Garamotte) [inline printer_copy_number/views/res_users.xml:3]: Replace `openerp` and `data` nodes by a single `odoo` node.

**Review by Garamotte — CHANGES_REQUESTED** [review]
Please add some unit tests and a `README.rst`.

Travis errors:
```
printer_copy_number/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
printer_copy_number/__manifest__.py:3: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file
```

**Comment by JosDeGraeve** [comment]
@jeanpaulrobineau can you take another look at this PR ?

**Pull request closed without merging**